pkg/kubelet/kubelet_node_status.go

```diff
@@ -273,6 +273,23 @@ func (kl *Kubelet) initialNode() (*v1.Node, error) {
 			glog.Infof("Adding node label from cloud provider: %s=%s", metav1.LabelInstanceType, instanceType)
 			node.ObjectMeta.Labels[metav1.LabelInstanceType] = instanceType
 		}
+
+		labels, err := instances.NodeLabels(kl.nodeName)
+		if err != nil {
+			return nil, err
+		}
+		if labels != nil {
+			prefix = "cloud.alpha.kubernetes.io/"
+			for name, value := range labels {
+				key := prefix + name
+				if len(key) > 63 {
+					glog.Warningf("Key: %s is too long, skipping", key)
```

Comment:
this appears to only happen once, so maybe spam is not a concern.  but then what happens if i tag my instance?  who gets it back on my node?  same with when i remove the tag on my instance, who removes it from my node.  feels dangerous if people start to depend on vm tagging schemes to drive behaviors for them to get out of synch absent reconciliation.